### ISSUE TITLE
Logging on 429 errors from Elasticsearch

### ISSUE THREAD

**Issue by jarro2783** [logstash-plugins/logstash-output-elasticsearch#745]
- Version: 9.0.2

When being rate limited by Elasticsearch there is a large volume of noisy INFO messages.

It looks like [they are written as debug logs][1], but [they are written on another line][2] as info logs. The message looks something like:
```
{"level":"INFO","loggerName":"logstash.outputs.elasticsearch","timeMillis":1520550508508,"thread":"Ruby-0-Thread-44@[main]>worker2: /usr/share/logstash/logstash-core/lib/logstash/pipeline.rb:384","logEvent":{"message":"retrying failed action with response code: 429 ({\"type\"=>\"es_rejected_execution_exception\", \"reason\"=>\"rejected execution of org.elasticsearch.transport.TransportService$7@784d57e7 on EsThreadPoolExecutor[bulk, queue capacity = 200, org.elasticsearch.common.util.concurrent.EsThreadPoolExecutor@501d117a[Running, pool size = 4, active threads = 4, queued tasks = 200, completed tasks = 131751374]]\"})"}}
```

This doesn't seem desirable to me. If you want to reduce logging of expected errors, they probably shouldn't end up logged somewhere else too.

[1]: https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/7dde2de9bdc48d4b5d80d63f905f2b3b989831e8/lib/logstash/outputs/elasticsearch/common.rb#L262
[2]: https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/7dde2de9bdc48d4b5d80d63f905f2b3b989831e8/lib/logstash/outputs/elasticsearch/common.rb#L192

**Comment by jsvd** [comment]
can you try setting `failure_type_logging_whitelist => [429]` ?

**Comment by jarro2783** [comment]
Yes that disables it. But I still think it is a bug that it is logged at all, given that there is code that deliberately ignores it.
